packages/rabbitmq/test/TxSubmitWorker.test.ts

```diff
@@ -76,8 +79,13 @@ describe('TxSubmitWorker', () => {
     await listenPromise(mock, port);
 
     worker = new TxSubmitWorker({ rabbitmqUrl: BAD_CONNECTION_URL }, { logger, txSubmitProvider });
+    const emitEventSpy = jest.spyOn(worker, 'emitEvent');
 
-    await expect(worker.start()).rejects.toBeInstanceOf(ProviderError);
+    try {
+      await expect(worker.start()).resolves.toBeUndefined();
```

Comment:
`expect` in both try and catch is confusing
```suggestion
      await worker.start();
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/344/commits/a84f4e1b695d3c14e0231a2170e67e0f65db966b I think now better explains the behaviour, with both resolving and emitting event